### DISCUSSION TITLE
也是进来了

### DISCUSSION THREAD

**Discussion by aaajszhjzbsjhzh** [share121/inter-knot#3144] [其他]
有没有好的手机加速器推荐，有点卡

**Comment by shan-dgha** [comment]
要什么样的，steam++完全够用<img src="https://bbs-static.miyoushe.com/static/2023/10/07/6feb23fa0de115295c11c3af4ab9a3b0_7714471584071518008.png" alt="卡琳_盯" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/10/07/4eb9952f04b4b5d48becd29cfbfd0f1c_3017775278024810598.png" alt="阿库_盯" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/10/07/3bf387f2e0ad67f34eec0baf2bc18326_1778988869407212201.png" alt="蒂亚拉_盯" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/10/07/bf273d925c48df58fea758bf8f598913_4888707432397159566.png" alt="拉妮_盯" width="75" >
